CHANGELOG.md

```diff
@@ -248,6 +248,8 @@ This document tracks pending changes to packages. It is facilitating the write-u
 
 - Added a `HoldingFactoryKey` data type which is used to key holding factories.
 
+- Added an enumeration data type `HoldingStandard` for referring to various holding standards.
```

Comment:
If we decide to keep the version where the holding standard is in the `InstrumentKey` then we should mention it here

Reply:
Agree. For now, I added that it is part of the `InstrumentKey`.